src/controllers/invoices.js

```diff
@@ -120,7 +123,7 @@ export default function () {
               ? items.filter(item => item.quantity > 0).reduce((trsStr, item, i) => trsStr + `
                 <tr>
                   <td>${(i + 1)}</td>
-                  <td>${item.name} (${item.sku})
+                  <td>${(item.picture && item.picture.small ? `<img class="img-product pr-2" src="${item.picture.small.url}">` : '<span class="img-product pr-2">Sem imagem</span>')}${item.name} (${item.sku})
```

Comment:
```suggestion
                  <td>${(item.picture && item.picture.normal ? `<img class="img-product img-fluid pr-2" src="${item.picture.normal.url}" width="70">` : '')}
                  ${item.name} (${item.sku})
```

Não está sendo salvo em tamanho `small` mais por padrão (não usamos no Storefront), também acho desnecessário escrever _sem imagem_, se fosse o caso teria que usar tradução pra enUS pelo menos também.